sklearn/metrics/cluster/supervised.py

```diff
@@ -459,7 +465,7 @@ def completeness_score(labels_true, labels_pred):
     return homogeneity_completeness_v_measure(labels_true, labels_pred)[1]
 
 
-def v_measure_score(labels_true, labels_pred):
+def v_measure_score(labels_true, labels_pred, beta=1.0):
```

Comment:
We should update the documentation to include `beta`:

```
v = (1 + beta) * (homogeneity * completeness) / (homogeneity + completeness)
```

Reply:
Sure @thomasjpfan ! Just added it